client/my-sites/stats/activity-log/index.jsx

```diff
@@ -5,6 +5,7 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import { groupBy, map } from 'lodash';
+import moment from 'moment';
```

Comment:
The `localize` HoC will inject a `moment` instance as a [prop](https://github.com/Automattic/i18n-calypso#moment-method) so we don't need to import moment here